### DISCUSSION TITLE
Multiline env vars from examples don't work?

### DISCUSSION THREAD

**Discussion by tino** [huggingface/chat-ui#1406] [Q&A]
So the documentation shows me to put this in an `.env.file`:
```
MODELS=`[
  {
      "name": "claude-3-haiku-20240307",
      "displayName": "Claude 3 Haiku",
      "description": "Fastest and most compact model for near-instant responsiveness",
      "multimodal": true,
      "parameters": {
        "max_new_tokens": 4096,
      },
      "endpoints": [
        {
          "type": "anthropic",
          "apiKey": "sk-ant-...",
          "baseURL": "https://api.anthropic.com",
          "defaultHeaders": {},
          "defaultQuery": {}
        }
      ]
  }
]`
```

But when I that Docker says: `docker: poorly formatted environment: variable '"name": "claude-3-haiku-20240307",' contains whitespaces.`

I can get it to run by replacing all newlines with `\n`, but that is certainly not stated anywhere as being needed. I even updated to the last docker engine, to see if I was missing something?

**Comment by nsarrazin** [comment]
Yeah this is a known issue with docker parsing the .env could you try the fixes suggested in this issue and let me know if that answered your question? https://github.com/huggingface/chat-ui/issues/1436
